sov-rollup-starter/README.md

```diff
@@ -1,5 +1,27 @@
-# Rollup Template
 
-This is still work in progress, please don't use it.
+# This is still work in progress. 
 
-This package is a convenient starting point for building a rollup using the Sovereign SDK
+This package is a convenient starting point for building a rollup using the Sovereign SDK:
+
+
+# How to run the sov-rollup-starter:
```

Comment:
Because of the Makefile issue, this tutorial currently fails. I suggest we either add `bashtestmd` checks in CI, or we restore the warning "please don't use it", possibly making it even more prominent to let users know that we're aware the tutorial is broken. (IMO this tutorial should be an HTML comment within the `README` with a `FIXME` notice unless fixed in this PR. Your call)

Reply:
```
sov-rollup-starter readme will be extended in the next PRs (this crate will be extracted to a separate repo)

```

I still keep the crate in this repo to check if all the import work as intended. In the next PR it will be extracted to a separate repo with its own CI and `bashtestmd `

Reply:
I missed that part, sorry! Makes sense.